src/app/learn/reference/template-items/index.html

```diff
@@ -35,7 +35,7 @@
     </li>
     <li>
       <p>
-        <stache-code>coverage</stache-code> folder — Stores code coverage results for unit tests. This folder is generated when you run <stache-code>skyux test</stache-code> or <stache-code>skyux watch</stache-code>, and it provides a convenient way for developers to see what code the tests reached and what code they did not reach. We recommend that all new code should have 100 percent unit test code coverage. For information about the code coverage requirements to contribute to SKY UX, see the <a href="https://github.com/blackbaud/skyux2/blob/master/CONTRIBUTING.md">SKY UX contribution guidelines</a>.
+        <stache-code>coverage</stache-code> folder — Stores code coverage results for unit tests. This folder is generated when you run <stache-code>skyux test</stache-code> or <stache-code>skyux watch</stache-code>, and it provides a convenient way for developers to see what code the tests reached and what code they did not reach. We recommend that all new code should have 100 percent unit test code coverage. For information about the code coverage requirements to contribute to SKY UX, see the <a routerLink="/contribute/guidelines">SKY UX contribution guidelines</a>.
```

Comment:
Super-trivial, but I'd either delete "to SKY UX" right before the link in the last sentence, or I'd delete "SKY UX" in the link text itself.

Reply:
It looks like the changes to this item were undone for some reason.

Reply:
I fixed the link locally and removed one of the redundant "SKY UX" instances in close proximity. I'll push those changes up to the PR.